source/_integrations/palazzetti.markdown

```diff
@@ -56,3 +70,57 @@ The Palazzetti integration offers the following sensors, for the products that p
 - Hydro temperature 2 (°C)
 - Pellet quantity (kg)
 - Pellet level (cm)
+
+## Possible use-cases
+
+- Control the operations, temperature, fans
+- Get alerts when the pellet level is low or empty, or on stove errors
+- Auto start or stop the stove based on presence
+
+## Automations
+
+Get started quickly with these automation examples.
+
+### Automatically lower the temperature when the last person leaves home
+
+{% details "Example YAML configuration" %}
+
+{% raw %}
+
+```yaml
+alias: Lower the temperature when last person leaves
```

Comment:
Please make sure examples follow our YAML style guide as specified here:

<https://developers.home-assistant.io/docs/documenting/yaml-style-guide>

Additionally, let's remove defaults from the example.